src/connection-manager/manifests.ts

```diff
@@ -96,9 +99,13 @@ export async function updateUNLManifests(): Promise<void> {
  * @param network - The network to update.
  * @returns A promise that resolves to void once all UNL validators are saved.
  */
-async function updateUNLManifestNetwork(network: string): Promise<void> {
+export async function updateUNLManifestNetwork(network: string): Promise<void> {
+  // TODO: In an ideal scenario, the networks table should not contain undefined network.id rows
```

Comment:
network.id will never be undefined since it's the primary key, so insert would fail if id is not set